bessctl/conf/testing/test_constraint_checker.bess

```diff
@@ -0,0 +1,67 @@
+
+def queue_test():
```

Comment:
Perhaps we don't need `queue_test()` and `nat_test()` here, since they are already checked in `samples/nat.bess` and `sample/queue.bess`and they are already included in the sanity check?